src/PsrLoggerAdapter.php

```diff
@@ -67,16 +63,17 @@ public function getLogger()
      * @param mixed  $level
      * @param string $message
      * @param array  $context
-     * @return void
      * @throws InvalidArgumentException If log level is not recognized.
      */
-    public function log($level, $message, array $context = [])
+    public function log($level, $message, array $context = []): void
```

Comment:
Let's please not add type information where it would break subclasses

Reply:
I had to add this because  this was enforced by in psr/log 3.0 
https://github.com/php-fig/log/blob/3.0.0/src/LoggerTrait.php

Reply:
Also, I was thinking that this was just a wrapper to offer compatibility from laminas/log to psr/log ... most likely it was used as is, but you are right, it can break sub classes..

Reply:
Does it make sense to put additional effort into this (make different branch for targeting different psr/log versions)?

Reply:
I think this component should really not be touched: moving to `Monolog\Monolog` is endorsed.

Reply:
> I think this component should really not be touched: moving to `Monolog\Monolog` is endorsed.

Understoo, thank you.